pages/helper/match_algo.py

```
@@ -50,78 +35,99 @@ def get_registered_cases_data(status="NF"):
+    except Exception:
+        return None
+
+
+# ---------------- CORE MATCHING ---------------- #
 
+def match(similarity_threshold=0.50):
+    """
+    Realistic thresholds:
+        0.75+ → FaceID / Banking
+        0.60+ → Corporate
+        0.45–0.55 → Police / Missing persons (YOU)
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)